raiden/network/protocol.py

```diff
@@ -327,7 +329,7 @@ def healthcheck(
             data,
             receiver_address,
             event_stop,
-            [nat_keepalive_timeout] * nat_keepalive_retries,
+            [DEFAULT_INITIAL_HEALTHCHECK_TIMEOUT] * DEFAULT_INITIAL_HEALTHCHECK_RETRIES,
```

Comment:
There is no need for the new configuration variables, `nat_keepalive_timeout` and `nat_keepalive_retries` are set through the configuration variables `DEFAULT_NAT_KEEPALIVE_TIMEOUT` and `DEFAULT_NAT_KEEPALIVE_RETRIES` and are properly propagated from the ui to the healthcheck.